change/@fluentui-react-native-e2e-testing-93969873-950e-4c8e-addd-f384491bc7c1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add more baseline Menu E2E tests",
+  "packageName": "@fluentui-react-native/e2e-testing",
+  "email": "winlarry@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
`dependentChangeType` I believe this value still needs to be changed

Reply:
Will do.